dependencies.gradle

```diff
@@ -2,11 +2,14 @@ def versions = [
         androidPlugin      : "3.2.0",
         androidTools       : "26.2.0",
         butterKnife        : "9.0.0-SNAPSHOT",
-        dagger             : "2.15",
-        kotlin             : "1.2.61",
+        dagger             : "2.16",
```

Comment:
lets use 2.19

Reply:
We agreed to use 2.16 in person (just detailing here why the change wasn't made)